eng/pipelines/steps/APIScan.yml

```diff
@@ -8,14 +8,17 @@ steps:
     SourceFolder: ${{ parameters.FolderToScan }}
     Contents: |
       **\*Microsoft@(*.dll|*.pdb|*.exe)
-      **\*Newtonsoft@(*.dll|*.pdb|*.exe)
       **\*OpenDebugAD7@(*.dll|*.pdb|*.exe)
       **\*WindowsDebugLauncher@(*.dll|*.pdb|*.exe)
       !**\*.resources.dll
+      !**\Microsoft.VisualStudio.Debugger.Interop.15.0.dll
```

Comment:
>!**\Microsoft.VisualStudio.Debugger.Interop.15.0.dll [](http://example.com/codeflow?start=6&length=52)

Is this because the scan tool needs to find all the dependencies?

Reply:
This is to ignore dependent binaries since we should only be scanning what was created.

Reply:
Ah. Should we just ignore all interop dlls?

Reply:
All except for `Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.DesignTime.dll` 
15 is the only one dropped for now but I can update the expression to include UnixPortSupplier and no other Microsoft.VisualStudio.Debugger.Interop* like we do for the ones under `vscode` folder.

https://github.com/microsoft/MIEngine/tree/main/src/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier

Reply:
If it is easy to do, I think that would be a good idea since we will probably add additional interop dlls in the future and forget to update this list.